sites/next.skeleton.dev/src/content/docs/integrations/popover/react.mdx

```diff
@@ -0,0 +1,193 @@
+| `useListNavigation` | Adds arrow key-based navigation of a list of items, either using real DOM focus or virtual focus. | [View](https://floating-ui.com/docs/useListNavigation) |
+| `useTypeahead` | Provides a matching callback that can be used to focus an item as the user types, used in tandem with `useListNavigation()`. | [View](https://floating-ui.com/docs/useTypeahead) |
+| `useTransition` | Provides the ability to apply CSS transitions to a floating element, including correct handling of “placement-aware” transitions. | [View](https://floating-ui.com/docs/useTransition) |
+| `useClientPoint` | Positions the floating element at a given client point `(x, y)`, usually generated by a mouse event. By default, the client’s mouse position is automatically tracked. | [View](https://floating-ui.com/docs/useClientPoint) |
+| `FloatingArrow` | Renders a customizable `<svg>` pointing arrow triangle inside the floating element that gets automatically positioned. | [View](https://floating-ui.com/docs/FloatingArrow) |
+| `FloatingFocusManager` | Provides flexible modal or non-modal focus management for a floating element. Modal behavior is the default — focus is fully trapped inside the floating element while it is rendered. | [View](https://floating-ui.com/docs/FloatingFocusManager) |
+| `FloatingPortal` | Portals the floating element into a given container element — by default, outside of the app root and into the body. | [View](https://floating-ui.com/docs/FloatingPortal) |
+| `FloatingTree` | Provides context for nested floating elements when they are not children of each other on the DOM. | [View](https://floating-ui.com/docs/FloatingTree) |
+| `FloatingOverlay` | Provides base styling for a fixed overlay element. | [View](https://floating-ui.com/docs/FloatingOverlay) |
+| `FloatingList` | Provides the ability to create composable children APIs for list components | [View](https://floating-ui.com/docs/FloatingList) |
+| `FloatingDelayGroup` | Provides context for a group of floating elements that should share a `delay` which temporarily becomes `1` ms after the first floating element of the group opens. | [View](https://floating-ui.com/docs/FloatingDelayGroup) |
+| `Composite` | Creates a single tab stop whose items are navigated by arrow keys, which provides list navigation outside of floating element contexts. | [View](https://floating-ui.com/docs/Composite) |
+| `Inner` | Positions the floating element such that an inner element inside of it is anchored to the reference element, while handling scroll and overflow behavior. | [View](https://floating-ui.com/docs/inner) |
+| `React Utils` | Merges an array of refs, returning a memoized callback `ref` or `null`. | [View](https://floating-ui.com/docs/react-utils) |
+| `Custom Hooks` | You can build your own custom Hook to perform unique logic that is not currently exported by the library. | [View](https://floating-ui.com/docs/custom-hooks) |
```

Comment:
I love this list. I'd love it a hell of a lot more if it were maintained by floating ui and not us though.

Reply:
As mentioned in my other comment, this is straight from them. It hasn't changed in the many months I've been reviewing Floating UI, so I think we're ok to keep this local copy for quick reference. But yeah, likewise I wish we could just link to a page that included all this for us!